### PR TITLE
feat: Add Try it buttons

### PR DESCRIPTION
**Pull request by srambach**
This adds Try it buttons to the Data Science and Overview page as specified in https://issues.redhat.com/browse/RHODS-3174 and https://issues.redhat.com/browse/RHODS-3175.
Also changes the ‘Beta’ button to ‘Field trial’ on Red Hat OpenShift Data Science tile on the overview page.

### CODE REVIEW

**Review by dlabaj — CHANGES_REQUESTED** [review]
Changes look good looks like you have to update the snapshots and make sure you run linting.

File: src/AppServices/DataSciencePage/DataSciencePage.tsx [hunk 1]
```diff
@@ -29,17 +30,29 @@ export const DataSciencePage: FunctionComponent = () => {
...
+            ouiaId="button-rhoads-tryit"
             variant={ButtonVariant.secondary}
             component="a"
-            href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science"
+            href="https://developers.redhat.com/products/red-hat-openshift-data-science/getting-started"
             target="_blank"
           >
-            {t("datascienceoverview:heroCallToActionButton")}{" "}
-            <ExternalLinkAltIcon className="pf-u-ml-sm" />
+            {t("datascienceoverview:heroTryItButton")}{" "}
```
Comment (riccardo-forina) [inline src/AppServices/DataSciencePage/DataSciencePage.tsx:42]:
```suggestion
            {t("datascienceoverview:heroTryItButton")}
```

Is the space needed?

**Review by dlabaj — APPROVED** [review]
LGTM

File: src/AppServices/DataSciencePage/DataSciencePage.tsx [hunk 2]
```diff
@@ -29,16 +30,28 @@ export const DataSciencePage: FunctionComponent = () => {
...
-            href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science"
-            target="_blank"
-          >
-            {t("datascienceoverview:heroCallToActionButton")}{" "}
-            <ExternalLinkAltIcon className="pf-u-ml-sm" />
-          </Button>
+          <Flex>
+            <Button
+              data-testid="hero-buttonTryIt"
+              ouiaId="button-rhoads-tryit"
```
Comment (riccardo-forina) [inline src/AppServices/DataSciencePage/DataSciencePage.tsx:36]: We are trying to have tests that explicitly check for changes of OUIA ids whenever we add one. Would you like trying adding one for this? It should be something along the lines of [this test](https://github.com/redhat-developer/app-services-ui-components/blob/dbb11acf6e58ae5a5e3d70b708db43a5e37d4686/src/Kafka/CreateKafkaInstance/CreateKafkaInstance.test.tsx#L9)

**Review by riccardo-forina — APPROVED** [review]
Fantastic, thank you for your help!

**Pull request merged**